### PR TITLE
Fix agg post agg

### PR DESCRIPTION
**Pull request by cosmic-chichu**
Aggregators Fixed:
- quantilesDoublesSketch

PostAggregators Fixed:
- quantileFromTDigestSketch
- quantilesDoublesSketchToHistogram
- quantilesDoublesSketchToQuantile
- quantilesDoublesSketchToQuantiles
- quantilesFromTDigestSketch

### CODE REVIEW

**Comment by cosmic-chichu** [comment]
@jbguerraz please review

File: builder/postaggregation/post_aggregator.go [hunk 1]
```diff
@@ -60,12 +61,19 @@ func Load(data []byte) (builder.PostAggregator, error) {
...
 		p = NewQuantilesDoublesSketchToQuantile()
+	case "quantilesDoublesSketchToQuantiles":
+		p = NewQuantilesDoublesSketchToQuantiles()
+	case "quantilesDoublesSketchToHistogram":
+		p = NewQuantilesDoublesSketchToHistogram()
 	default:
 		return nil, errors.New("unsupported postaggregation type")
 	}
-	return p, json.Unmarshal(data, &p)
+	return p, json.Unmarshal(data,
```
Comment (jbguerraz) [inline builder/postaggregation/post_aggregator.go:77]: Does that pass the go lint ?

**Review by jbguerraz — COMMENTED** [review]
LGTM.
Some formatting looked a bit weird to me,did it pass thru go lint ?
other than that, it's good to go :) thank you!

File: builder/postaggregation/post_aggregator.go (hunk 1 above)
Comment (cosmic-chichu) [reply builder/postaggregation/post_aggregator.go:77]: yes it does, it's my IDE that formatted it differently. I'll fix it :)

**Comment by vigith** [comment]
@cosmic-chichu can you please paste the lint result?

**Comment by cosmic-chichu** [comment]
(⎈ |k3d-k3s-default:default)➜  go-druid git:(fixAggPostAgg) golangci-lint run 
builder/query/query.go:66:54: Error return value of `(*github.com/grafadruid/go-druid/builder/datasource.Query).UnmarshalJSONWithQueryLoader` is not checked (errcheck)
                        d.(*datasource.Query).UnmarshalJSONWithQueryLoader(tmp.DataSource, Load)
                                                                          ^
druid.go:256:9: Error return value of `io.Copy` is not checked (errcheck)
        io.Copy(ioutil.Discard, io.LimitReader(resp.Body, respReadLimit))
               ^
builder/types/join_types.go:6:2: SA9004: only the first constant in this group has an explicit type (staticcheck)
        Inner JoinType = "INNER"
        ^
builder/types/null_handling.go:6:2: SA9004: only the first constant in this group has an explicit type (staticcheck)
        NullString  NullHandling = "NULLSTRING"
        ^
builder/types/output_types.go:6:2: SA9004: only the first constant in this group has an explicit type (staticcheck)
        Float   OutputType = "FLOAT"
        ^


(⎈ |k3d-k3s-default:default)➜  go-druid git:(fixAggPostAgg) golangci-lint run --fast
(⎈ |k3d-k3s-default:default)➜  go-druid git:(fixAggPostAgg) echo $?
0
(⎈ |k3d-k3s-default:default)➜  go-druid git:(fixAggPostAgg)

**Review by vigith — APPROVED** [review]

**Pull request merged**